run-tests.sh

```diff
@@ -6,11 +6,17 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-python setup.py compile_catalog && \
-pybabel compile -d tests/translations/ && \
-pydocstyle invenio_i18n && \
-isort -rc -c -df **/*.py && \
-check-manifest --ignore ".travis-*" && \
-sphinx-build -qnNW docs docs/_build/html && \
-python setup.py test && \
-sphinx-build -qnNW -b doctest docs docs/_build/doctest
+# Quit on errors
+set -o errexit
+
+# Quit on unbound symbols
+set -o nounset
+
+python setup.py compile_catalog
+pybabel compile -d tests/translations/
```

Comment:
Is there a way of invoking `pybabel` and `compile_catalog` the `python -m` way? We should try to always do it for tests reproducibility.

Reply:
`compile_catalog` is executing by `python -m` not `pybabel`.

Reply:
Don't they do the same? both seem to endup [here](https://github.com/python-babel/babel/blob/0a3402594d66eb4c8f76c42563a91a8ec88c3b04/babel/messages/frontend.py#L129)

Reply:
When I said invoking `pybabel` and `compile_catalog` the `python -m` way I meant:
```diff
- python setup.py compile_catalog
+ python -m setup compile_catalog
- pybabel compile -d tests/translations/
+ python -m babel.messages.frontend compile -d tests/translations/
```
So all commands are called inside the virtualenv.

Reply:
@diegodelemos I understood, then I wondered if it was needed. Cuz they both seem to end up calling the same function 🤔. But anyway let's leave it as it was (both calls), with `python -m...` if possible.

Reply:
@ppanero my message was addressing Parth's comment https://github.com/inveniosoftware/invenio-i18n/pull/93#discussion_r537406386, however, you posted while I was writing! I agree, if the commands are doing exactly the same, please remove one of them.

Reply:
Concurrency issues 😅 I'd say:

- Leave it as is now.
- Open an issue to investigate the possibility of removing one of the two commands. In order not to overlad @lnielsen right now.

@ParthS007  WDYT? 
@diegodelemos If this is also fine with you, feel free to merge.

Reply:
Okay, I have opened another [issue](https://github.com/inveniosoftware/invenio-i18n/issues/95) and we can go ahead with merging this one.